src/sass/claims-status.scss

```diff
@@ -663,10 +665,21 @@ h1:focus {
 }
 
 .current-status {
+  &.closed {
+    .current-status-content {
+      border-left-color: transparent;
+
+      &::before {
+        background-color: $color-inactive;
```

Comment:
TIL we have a `$color-inactive`

Reply:
Nope. I made it a thing in this file only because I couldn't find that exact color code anywhere else. Not sure what the best practice here is. I figure we've got three ways to deal with it:
1. Change the design so the color matches an existing color
2. Add the color to our existing color variable collection in `_b-variables.scss` (or whatever it is)
3. Add it to just this file if it's not going to be used anywhere else

I just went with the last option. Dunno if that was right. :man_shrugging: